content/_index.md

```diff
@@ -24,4 +24,189 @@ card2:
+servicesHead:
+  title: Professional Psychology Therapy Service You Can Choose
+  description: affordable Services
+servicesCard1:
+  title: Couple Counseling
+  description: Nostrud exerctation to ullamco laibor nisiod in above murlialiquip commodo
+  button: Read details
+servicesCard2:
+  title: Couple Counseling
+  description: Nostrud exerctation to ullamco laibor nisiod in above murlialiquip commodo
+  button: Read details
+servicesCard3:
+  title: Couple Counseling
+  description: Nostrud exerctation to ullamco laibor nisiod in above murlialiquip commodo
+  button: Read details
+servicesCard4:
+  title: Couple Counseling
+  description: Nostrud exerctation to ullamco laibor nisiod in above murlialiquip commodo
+  button: Read details
+servicesCard5:
+  title: Couple Counseling
+  description: Nostrud exerctation to ullamco laibor nisiod in above murlialiquip commodo
+  button: Read details
+servicesCard6:
+  title: Couple Counseling
+  description: Nostrud exerctation to ullamco laibor nisiod in above murlialiquip commodo
+  button: Read details    
+
+projectArea: 
+  span: Latest Projects
+  title: Focusing Your Mind, With The Best Therapeutic Care
+
+projectArea1: 
+  image: assets/img/project/project-1.jpg
+  span: Career Frustation
+  title: Personal Meeting
+projectArea2: 
+  image: assets/img/project/project-2.jpg
+  span: Career Frustation
+  title: Consultation
+projectArea3: 
+  image: assets/img/project/project-3.jpg
+  span: Career Frustation
+  title: Couple Problem
+projectArea4: 
+  image: assets/img/project/project-4.jpg
+  span: Career Frustation
+  title: Depression
+projectArea5: 
+  image: assets/img/project/project-2.jpg
+  span: Career Frustation
+  title: Mental Problem
+
+teamArea:
+  title: Focusing Your Mind, With The Best Therapeutic Care
+  span: Expert member
+teamMember1:
+  image: assets/img/team/03/team-01.jpg
+  name: Donnie Anderson
+  span: Sr. Psychologist 
+  facebook: index-3.html#
+  twitter: index-3.html#
+  pinterest: index-3.html#
+  email: 123@gmail.com
+teamMember2:
+  image: assets/img/team/03/team-02.jpg
+  name: Tashanu Canalse
+  span: Therapy Expert 
+  facebook: index-3.html#
+  twitter: index-3.html#
+  pinterest: index-3.html#
+  email: 254@gmail.com
+teamMember3:
+  image: assets/img/team/03/team-03.jpg
+  name: Richard Sikora
+  span: CEO & Founder
+  facebook: index-3.html#
+  twitter: index-3.html#
+  pinterest: index-3.html#
+  email: sync@gmail.com
+
+counterArea1:
+  numbers: 2950
+  span: Psychologist
+counterArea2:
+  numbers: 1520
+  span: All Branches
+counterArea3:
+  numbers: 3250
+  span: Total patients
+counterArea4:
+  numbers: 2150
+  span: Achievements
+
+whyChooseArea:
+  span: Why choose us
+  title: Best Method Apply our Patients
+  list1: Psychodynamic Therapy
+  list2: Personal Development
+  list3: Motivational Speech
+  list4: Mindtherafy Professional
+  list5: Assesment Consultation
+  image: assets/img/why/why-img-03.jpg
+
+imageTestimonialArea:
```

Comment:
all testimonials are of should be one design and already moved to partial use the same designed testimonial in homepage